src/main/resources/META-INF/rewrite/assertj.yml

```
@@ -45,6 +45,224 @@ recipeList:
       fullyQualifiedTargetTypeName: "org.assertj.core.api.Assertions"
   - org.openrewrite.java.UseStaticImport:
       methodPattern: "org.assertj.core.api.Assertions *(..)"
+
+  # Simplify Chained AssertJ Assertions
+  # String Assertions
+  - org.openrewrite.java.SimplifyChainedAssertJAssertion:
```

This and all below likely need the correct package & classname.
```suggestion
  - org.openrewrite.java.testing.assertj.SimplifyChainedAssertJAssertion:
```